README.md

```diff
@@ -1,5 +1,15 @@
 # boats.py
-The official discord.boats API wrapper for Python
```

Comment:
Maybe change it to The official discord.boats python library or something

Reply:
Sure. I just didn't want to mess with that line a lot since it's the description as well.